Core/GDCore/Project/EventsFunctionsExtension.cpp

```diff
@@ -138,11 +144,48 @@ void EventsFunctionsExtension::UnserializeFrom(
     dependencies.push_back(
         UnserializeDependencyFrom(dependenciesElement.GetChild(i)));
 
+  // Only unserialize behaviors and objects names.
+  // As event based objects can contains objects using CustomBehavior and/or
```

Comment:
When an object is created, the type is used to check if a custom object must be created or not. The event-based object needs to be declared in the project for this check.
Actually, it could work if we create a custom object by default when the type is unknown but it would be impossible to log a warning to differentiate it from the nominal case.